src/lib.rs

```diff
@@ -37,9 +37,9 @@ pub fn nif_ecrecover<'a>(env: Env<'a>, args: &[Term<'a>]) -> Result<Term<'a>, Er
     let mut byte_ref = Vec::new();
     let ecrecover = EcRecover { };
     let _result = match ecrecover.execute(input.as_slice(),
-                                          &mut BytesRef::Flexible(&mut byte_ref)) {
+                                          &mut BytesRef::Fixed(&mut byte_ref)) {
```

Comment:
It always fails with BytesRef::Fixed, even if I set the initial capacity to 65 (which is a good thing anyway), so changing back.